src/header.js

```diff
@@ -0,0 +1,45 @@
+import homePage from './home';
+import contactPage from './contact';
+import menuPage from './menu';
+
+export const renderSection = (section) => {
```

Comment:
- [ ] It would be great if you use ES6 syntax here. For example, don't export a function before defining a function.